### PR TITLE
Website publish requires issues:write

### PR DESCRIPTION
**Pull request by ab77**
Fixes: https://github.com/balena-io/balena-cloud/actions/runs/11783098446/job/32825587737#step:12:14

### CODE REVIEW

File: flowzone.yml [hunk 1]
```diff
@@ -3367,6 +3367,7 @@ jobs:
             {
               "metadata": "read",
               "contents": "read",
+              "issues": "write",
```
Comment (klutchell) [inline flowzone.yml:3370]: `"pull_requests": "write"` should be enough, based on testing I did elsewhere.

Comment (klutchell) [reply flowzone.yml:3370]: Is this not enough? https://github.com/balena-io/balena-cloud/blob/e0a5401062a8e8799227e9d8f231fbf1d3078366/.github/workflows/flowzone.yml#L21C1-L21C49

Comment (klutchell) [reply flowzone.yml:3370]: Hmm, it is using the [listComments](https://github.com/peter-evans/find-comment/blob/main/src/find.ts) REST endpoint which may need more than actually posting a comment. @ab77 why don't we try just giving it "issues": "read"?

Comment (ab77) [reply flowzone.yml:3370]: I just took the settings from [here](https://github.com/peter-evans/create-or-update-comment)

image

Comment (klutchell) [reply flowzone.yml]: I think try it with just `issues:read` and that is hopefully enough.

Comment (klutchell) [reply flowzone.yml]: Let's try this instead: https://github.com/product-os/flowzone/pull/1252

**Pull request closed without merging**